lib/widgets/change_report_button.dart

```diff
@@ -0,0 +1,33 @@
+import "package:flutter/material.dart";
+
+import "../theme/colors.dart";
+import "../utils/context_extensions.dart";
+
+class ChangeReportButton extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 16, horizontal: 12),
+      child: Column(
+        children: [
+          Text(context.localize.change_report_title),
+          const SizedBox(height: 8),
+          ElevatedButton(
+            onPressed: () {},
+            style: ElevatedButton.styleFrom(
+              backgroundColor: ColorsConsts.blueAzure,
+              padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 10),
+              shape: RoundedRectangleBorder(
+                borderRadius: BorderRadius.circular(8),
```

Comment:
It's not super important but you can extract some values to config. You can also unify border radius ( in config you use padding with value 10)